aws/data_source_aws_kms_public_key.go

```diff
@@ -75,7 +76,7 @@ func dataSourceAwsKmsPublicKeyRead(d *schema.ResourceData, meta interface{}) err
 	d.Set("arn", output.KeyId)
 	d.Set("customer_master_key_spec", output.CustomerMasterKeySpec)
 	d.Set("key_usage", output.KeyUsage)
-	d.Set("public_key", string(output.PublicKey))
+	d.Set("public_key", base64.StdEncoding.EncodeToString(output.PublicKey))
```

Comment:
Is this correct? If my understanding is correct `output.PublicKey` is already base64 encoded, so should it be returned as is?

Reply:
I think you're correct @chrisclayson -- from the AWS Go SDK Documentation https://docs.aws.amazon.com/sdk-for-go/api/service/kms/#GetPublicKeyOutput
> PublicKey is automatically base64 encoded/decoded by the SDK

Reply:
Ah ... OK, fair enough, I was reading the AWS API docs, didn't realise the Go SDK automatically decoded it. Thanks for clarifying! 👍🏻